src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.html

```diff
@@ -250,7 +250,9 @@ <h2 *ngIf="tutorParticipationStatus === TRAINED || tutorParticipationStatus ===
                             <td>
                                 <button
                                     *ngIf="exercise.type !== ExerciseType_PROGRAMMING; else programmingAssessment"
-                                    class="btn btn-success btn-sm"
+                                    class="btn btn-success btn-sm assessment-button"
+                                    [class.guided-tour]="isGuidedTourAssessment()"
+                                    [disabled]="isGuidedTourAssessment()"
```

Comment:
Can we not set it as a property once? Because in this case it will be called on every small cahnge